signalr-go-server/pkg/signalr/hubprotocol.go

```diff
@@ -18,11 +23,17 @@ type hubMessage struct {
 }
 
 type invocationMessage struct {
```

Comment:
Undo this

Reply:
01e83d66d85ae95bd8278ae2fdd940464c3253cd